common/CMakeLists.txt

```diff
@@ -1,21 +1,16 @@
 #Copyright (C) Microsoft Corporation. All rights reserved.
 
 set(pal_common_h_files
-    inc/gballoc.h
+    ../common/inc/gballoc.h
+    ../common/inc/constbuffer.h
+    PARENT_SCOPE

```

Comment:
>PARENT_SCOPE [](start = 4, length = 12)

why do we need this construct? Can it be avoided at all?

Reply:
This is needed so that the common files can be built with pal_win32 or pal_linux.

---
In reply to: [452444012](https://github.com/Azure/azure-c-pal/pull/7#discussion_r452444012) [](ancestors = 452444012)

Reply:
I moved the common files to separate lists in the win32 and linux CMakelists.txt

---
In reply to: [452468890](https://github.com/Azure/azure-c-pal/pull/7#discussion_r452468890) [](ancestors = 452468890,452444012)